frontend/integration-tests/views/crud.view.ts

```diff
@@ -166,3 +166,23 @@ export const emptyState = $('.cos-status-box').$('.text-center');
 
 export const errorMessage = $('.pf-c-alert.pf-m-inline.pf-m-danger');
 export const successMessage = $('.pf-c-alert.pf-m-inline.pf-m-success');
+
+export const clickListPageCreateYAMLButton = async() => {
+  const createDropdownIsPresent = await createItemButton.isPresent();
+  if (createDropdownIsPresent) {
+    await createItemButton.click();
+    await createYAMLLink.click();
+  } else {
+    await browser.wait(until.presenceOf(createYAMLButton));
+    await createYAMLButton.click();
+  }
+  browser.wait(until.and(untilNoLoadersPresent, until.presenceOf(element(by.cssContainingText('h1', 'Create')))));
+};
+
+export const createNamespacedResourceWithDefaultYaml = async function(resourceType: string) {
```

Comment:
nit:

```suggestion
export const createNamespacedResourceWithDefaultYAML = async function(resourceType: string) {
```

to match case with `clickListPageCreateYAMLButton`

Reply:
I Updated the name, but after my push, the LGTM label was removed automatically. Could you help to add the lable again? Sorry for the trouble. (Seems I could click "Commit suggestion" button directly to get your suggestion into the pr, right? not familiar that, if yes, I could try it next time.)